### PR TITLE
[Asset] getText() on documents should process the asset synchronously called from CLI

### PR DESCRIPTION
**Pull request by brusch**
Follow-up to #6015

### CODE REVIEW

**Comment by brusch** [comment]
@blankse can you confirm this works for you?

File: lib/Maintenance/Tasks/AssetDocumentConversionTask.php [hunk 1]
```diff
@@ -50,9 +51,15 @@ public function execute()
...
-                        $asset->getFullPath()));
+                    $this->logger->debug(sprintf('Processing document with ID %s | Path: %s', $asset->getId(), $asset->getFullPath()));
                     $asset->processPageCount();
+                    $asset->save();
+
+                    $searchEntry = Data::getForElement($asset);
+                    if ($searchEntry instanceof Data and $searchEntry->getId() instanceof Data\Id) {
+                        $searchEntry->setDataFromElement($asset);
+                        $searchEntry->save();
+                    }
```
Comment (blankse) [inline lib/Maintenance/Tasks/AssetDocumentConversionTask.php]: `$asset->save();` also generate the search entry, right?

Comment (blankse) [inline lib/Maintenance/Tasks/AssetDocumentConversionTask.php]:
```suggestion
```

Comment (brusch) [reply lib/Maintenance/Tasks/AssetDocumentConversionTask.php]: You're totally right, `save()` does already the job :)

**Pull request merged**